pyflakes/test/test_api.py

```diff
@@ -55,26 +55,50 @@ def __init__(self, lineno, col_offset=0):
 
 class SysStreamCapturing(object):
 
-    """Replaces sys.stdin, sys.stdout and sys.stderr with StringIO objects."""
+    """
```

Comment:
Any reason why the first line of the docstring isn't on this line too? PEP 257 recommends it be on one line (not that this project explicitly follows that).